src/main/java/org/fcrepo/transform/ldcache/FedoraEndpoint.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.transform.ldcache;
+
+import org.apache.marmotta.commons.http.ContentType;
+import org.apache.marmotta.ldclient.api.endpoint.Endpoint;
+
+/**
+ * @author acoburn
+ */
+public class FedoraEndpoint extends Endpoint {
```

Comment:
I'll update the javadoc with more information.

The `baseUrl` is just a namespace that is used by marmotta's `LDClient` to determine which `Endpoint` rules to use when dereferencing a given URI. This `baseUrl` effectively means this: whenever a URI starts with the specified value (e.g. http://localhost:8080/fcrepo/rest or http://repo.example.org/), then apply the settings of this `Endpoint` implementation.